helios-services/src/main/java/com/spotify/helios/master/reaper/OldJobReaper.java

```diff
@@ -79,6 +82,8 @@
   private final long retentionMillis;
   private final Clock clock;
 
+  private Set<JobId> jobsInDeploymentGroups;
```

Comment:
unfortunate that the structure of the superclass means you have to store this data in a class field rather than a local somewhere, but I suppose not using the structure imposed on you by the superclass would be more annoying